google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +184,18 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def update_schema():
```

Comment:
This method does not do any update.  It would be better to be called as `get_update_actions()` or `get_migrate_actions()`.

Reply:
Updated.